Update README.md (#1)
fs-webdev/clashjs: README.md

  * Turn into any of the four directions. (`north`, `east`, `south`, `west`).
  * Shoot. (`shoot`).
* A player can shoot to try to destroy another player.
* A player can collect ammo in the moment it enters the square containing the ammo. New ammo may appear in any moment of the game. If ammo appears on your square, you will not collect it unless you move off and back on.

**Note**: Please do not try and hack or break the game. There are ways to cheat and modify the game state or other players (not through the objects given to your `ai` function, but other means). We will try to review the bots as you PR them in, but please just stay within the rules.

**Note**: We also ask that your team do all of your own coding. We have given you a basic starter bot and a helpers library with various helper functions. Please do not go find code from other bots and copy them into your bot code.

## Running/Debugging

> A game consistes of several rounds. When a round completes, another round immediately starts. Once all of the rounds are complete, the game is over and the stats modal appears with the winner and final stats. To play another game, you have to reload the browser tab.

Reviewer: ```suggestion
A game consists of several rounds. When a round completes, another round immediately starts. Once all of the rounds are complete, the game is over and the stats modal appears with the winner and final stats. To play another game, you have to reload the browser tab.
```

```js
{
  info: {
    name: 'starterbot',

>     style: 2, // one of the 111 styles (0 to 110) see Rocket icon in game for list of styles

Author: ```suggestion
    style: 20, // one of the 111 styles (0 to 110) available. Click the Rocket icon in game for a catalog of all of the styles
```

    team: 1, // team number assigned to your team
  },
  ai: function(player, enemies, game) {
    // think...
    return 'move'; // or some other action
  }
}
```

The AI function will receive [`player`](#player-state), `enemies` (array of all the other players `playerState`s), and [`game`](#game-environment) as arguments, and must return one of the following strings:
  * `move`: To move one tile in the current direction.
  * `north`, `east`, `south` or `west`: To turn to that direction.
  * `shoot`. To shoot if the user has enough ammo.

Any other response, trying to move outside the arena size (`game.gridSize`) or trying to shoot without ammo, will result in a no-op.

All positions in the game are in a 2 item array: `[verticalOffset, horizontalOffset]` from upper left corner, zero indexed, so essentially `[Y, X]` except in the 4th quadrant so Y increases as you go down the board.

### Player State.

> Let the *player state* (`playerState`) be an object with a player information like the following:

Reviewer: ```suggestion
The *player state* (`player`) is an object containing the following information:
```
